src/content/deadlines/03_finaid.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Financial Aid
+subtitle: Open until 9 May
```

Comment:
```suggestion
subtitle: Open until May 9th
```

Reply:
Oh is there any reason you prefer the US date format here? I feel like we've been mostly sticking with the British/European format in other pages, so wanted to check.

Reply:
No preference, this just felt right instinctively. We can stick with the European way 🙂

Reply:
haha let's stick with the European way for this to satisfy my compulsion  :grin: